src/main/resources/static/openapi/EducationAndWorkPlanAPI.yml

```diff
@@ -973,15 +973,15 @@ components:
           type: array
           description: Reasons the Prisoner has given not to get work after leaving Prison. Can be empty but not null.
           items:
-            $ref: '#/components/schemas/ReasonNotToWork'
+            $ref: '#/components/schemas/NotHopingToWorkReason'
         notHopingToWorkOtherReason:
           type: string
           description: The reason that is given when the Prisoner does not want to work. This is mandatory when 'notHopingToWorkReasons' includes 'OTHER'.
         affectAbilityToWork:
           type: array
           description: Factors affecting the Prisoner's ability to work. Can be empty but not null.
           items:
-            $ref: '#/components/schemas/AbilityToWorkFactor'
+            $ref: '#/components/schemas/AffectAbilityToWork'
```

Comment:
I've added new enums which have the same names as the ones in our domain/entity model. We will remove the old CIAG ones when we eventually migrate from their API.

Reply:
Not sure I follow - are these part of the new API classes, or the original CIAG Induction classes?
Why did you need to create duplicates of them?

Reply:
Ah yes, I wasn't thinking! Of course I can just rename the existing enums and it wouldn't matter, so long as the field names are the same! 🤦‍♂️

Reply:
mmm...on second thoughts, it breaks more than I expected, as I'd have to rename mapper methods and all kinds of other things.

I think it would be much easier to these 2 new enums and then remove the old ones when we finally nuke the CIAG API.

Reply:
I still dont follow .... why do we need duplicates?

Reply:
As discussed on Slack, I'm adding two new enums until we nuke the CIAG API 👍